django/thunderstore/repository/admin/package_version.py

```diff
@@ -73,6 +82,15 @@ def file_tree_link(self, obj):
         )
         return mark_safe(f'<a href="{url}">{obj.file_tree}</a>')
 
+    file_tree_link.short_description = "File tree"
+
+    def listings(self, obj):
+        listings = PackageListing.objects.filter(package__versions=obj)
+        links = ""
+        for listing in listings:
+            links += f'<a href="{listing.get_admin_url()}">{listing.community}</a><br/>'
+        return mark_safe(links)
```

Comment:
Not the ideal way to build HTML but good enough. The bigger problem here is that this doesn't scale well if/when a package has too many listings, but that's unlikely to be a problem anytime soon so this is good enough for now.

The way I'd have approached this is to link to the package listing list page with a filter pre-applied in the query string that only selects listing objects for this package version, although that would be slightly more complicated to implement.

Reply:
I've applied the suggested change in 449113ecd5c1826485044019ff62cc2789bd2a50